src/mdmerge/main.cpp

```diff
@@ -0,0 +1,458 @@
+#include <cstdlib>
```

Comment:
Can you file an issue to remove the redunency between `mdmerge` and `mddump`? This logic has gone through fuzzing and I'd like to make sure we don't drift.

Reply:
Opened #37.